src/controllers/request-controller.ts

```diff
@@ -63,17 +63,20 @@ export default class RequestController {
     try {
       logger.debug(`Create request ${JSON.stringify(req.body)}`);
 
-      const { cid, docId } = req.body;
+      const cid = req.body.cid;
+      // TODO docId check for backwards compat with old Ceramic nodes. Remove once all infra nodes
+      // updated to new Ceramic version
```

Comment:
We should probably keep this around for longer. Ppl are using their own nodes on clay. Not only tied to our infra.
```suggestion
      // TODO docId check for backwards compat with old Ceramic nodes. Remove once confident that
      // most nodes in the network updated to new Ceramic version
```

Reply:
Please also indicate which version here for future reference.

Reply:
I won't know the version number to use before merging this.  I need to get this merged, released, and deployed on our infra, then I can make the change to ceramic to use the new format and get that released.

I'll include a reference to the ceramic github issue that will handle the support, that seems like the best we can do

Reply:
I guess you could just include the js-ceramic version we are currently at? Or better, a reference to the issue for fixing it!

Reply:
Ah, that's what you did 👍